src/main/controllers/features/common/typeOfBusinessController.ts

```diff
@@ -93,7 +93,7 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
             };
             // calling  postAcspRegistration api
             try {
-                const acspResponse = await postAcspRegistration(session, session.getExtraData(SUBMISSION_ID)!, acsp);
```

Comment:
This can be deployed with feature flagging, so that only staging will have this change enable.